### DISCUSSION TITLE
How to do data lineage analysis for data processed under seatunnel?

### DISCUSSION THREAD

**Discussion by tmljob** [apache/seatunnel#1483] [Q&A]
How to do data lineage analysis for data processed under seatunnel? Is there any case or plan to refer to?

**Comment by Hisoka-X** [comment]
In my view, seatunnel focuses on data synchronization rather than data analysis.

**Comment by tmljob** [comment]
> In my view, seatunnel focuses on data synchronization rather than data analysis.


Yes, what I mean is how to analyze the blood relationship of these data after ETL data processing through Seatunnel? Is it done by analyzing the configuration file of the task? Has anyone done similar work to learn from?

**Comment by Hisoka-X** [comment]
Seem like need to parse the configuration file

**Reply by lphcreat** [reply]
i wrote a script,for parse conf getting lineage,you can use it.https://github.com/lphcreat/parse_lineage.git
